src/coreclr/vm/typeparse.cpp

```diff
@@ -3,1172 +3,69 @@
 // ---------------------------------------------------------------------------
 // typeparse.cpp
 // ---------------------------------------------------------------------------
```

Comment:
The diff for this file is confusing. It is best to just look at the new file. There is not much left in it.